test/e2e/specs/components/SelectedLicense.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+	beforeEach : function(browser) {
+		const serverUrl = 'http://localhost:8080/';
```

Comment:
Please do not hard code the server URL. Take a look at other e2e specs to see how they do it :)